src/translations/loader.ts

```diff
@@ -0,0 +1,13 @@
+import { Strings } from 'lit-translate';
+import { de } from './de.js';
+import { en } from './en.js';
+
+export type Language = 'en' | 'de';
+export const languages = { en, de };
+
+export type Translations = typeof en;
```

Comment:
missed 'de' ?

Reply:
nope it's just a way to extract the typing inside of the Language object, en is the "root"